README.md

```diff
@@ -0,0 +1,69 @@
+![](https://img.shields.io/badge/Microverse-blueviolet) 
+
+# Web Summit Conference: Mobile and Desktop Version
+
+> This is a HTML/CSS and Javscript based conference web application.it has been developed as part of a Microverse capstone project.
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Flexbox
+- Linters (Lighthouse, Webhint, Stylelint, Eslint)
+- Visual Studio Code
+- Git & Github
+
+## Live Demo
+
+- [App Live Demo](https://chimwemwe007.github.io/CAPSTONE-MODULE1-HTML-CSS-Javascript/)
+
+
+## Project Description
+- [Description](https://www.loom.com/share/84ed9dafd9c34dbca657de1a67012c40)(This project is a website for a summit centred around shedding awareness on underpriviledged and orphaned children.)
```

Comment:
- [ OPTIONAL] Your project demo link is not working, please fix the issue so that you can run the project successfully using the project link
- [ ] As previous reviewer suggested you need to have a recorded videos that summarize the features that you've been working on. here is the criteria for the videos 
![image](https://user-images.githubusercontent.com/43877689/175788289-0c6946dd-2d65-4a8d-b173-5d5e70efabf5.png)

Reply:
Hello i included a link to my loom video already in the comments. as well as my readme file..

Reply:
Hello @ishimwezachee 

I have fixed the deployed version. Please also find the loom video both in the PR description and README file. Thanks